src/matrix/room/timeline/Timeline.js

```diff
@@ -141,6 +141,7 @@ export class Timeline {
             // redactingEntry might be a PendingEventEntry or an EventEntry, so don't assume pendingEvent
             const relatedTxnId = pee.redactingEntry.pendingEvent?.relatedTxnId;
             this._findAndUpdateEntryById(relatedTxnId, pee.redactingEntry.relatedEventId, updateOrFalse);
+            pee.redactingEntry.contextForEntries?.forEach(e => this._emitUpdateForEntry(e));
```

Comment:
the reason `e` is emitting an update here is because it's contextEntry might have changed. Should we add a `param` parameter to `_emitUpdateForEntry` as an argument to the update event so we can specify `"contextEntry"` here (and elsewhere where we call `_emitUpdateForEntry`)?

Reply:
yes, we do that for updates everywhere else 👍